.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,23 @@
+Issue: https://issues.redhat.com/browse/PROJQUAY-???
+Pull-request title must start with "PROJQUAY-??? - "
 
-### Description of Changes
```

Comment:
Why change these from headings?

Reply:
As discussed, keeping them as headers just seemed too large. Wanting to keep this simple (and simple looking). Bold was a good idea.